api/src/models/users.py

```diff
@@ -0,0 +1,42 @@
+from db import db
+
+class Users(db.Model):
+
+    __tablename__ = 'Users'
+
+    id = db.Column(db.Integer, primary_key=True)
+    first_name = db.Column(db.String(120), nullable=False)
+    is_bot = db.Column(db.Boolean, nullable=False)
+
+    def save(self):
+
+        try:
+            db.session.add(self)
+            db.session.commit()
+        except Exception as e:
+            return
+            
+    @classmethod
+    def retrieve_single_user(cls,id):
+        
+        return cls.query.get(id)
+    
+    @classmethod
+    def delete_user(cls, id): 
+        
+        with self.retrieve_single_user(id) as user:
+            db.session.delete(user)
+            db.session.commit()
+
+    @classmethod
+    def update_user(cls, id, name, is_bot):
```

Comment:
-> `def update()`

Also why pass the `name` here? 

And why do we delete the user and not update in place?